mlflow/tracking/fluent.py

```diff
@@ -302,8 +297,21 @@ def _get_experiment_id_from_env():
     return env.get_env(_EXPERIMENT_ID_ENV_VAR)
 
 
+def _get_or_create_experiment(experiment_name):
```

Comment:
What is the motivation to create a new method? Was it to use it in `_get_experiment_id` function?